portfolio/src/main/webapp/index.html

```diff
@@ -2,13 +2,15 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Andrew Pham | Home</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
+      <h1> Andrew Portfolio</h1>
+      <h3>Hello! My name is Andrew Pham and I am a second year at the University of California San Diego. I grew up with a family of six: two parents, an older brother and sister, <br>
```

Comment:
Instead of using an h3 here, prefer using a <p> tag. You can use CSS to change the font size if you want it to look big. The goal of HTML is to represent the structure of the code as well as possible, so you should use p (paragraph) tags when it is content, and h1-6 when it is a header of some kind

Reply:
Alright I see. I was mainly testing to see how a pull request works! I'll change the tags so that the description is in a <p> tag and use CSS to style the font.